packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -103,15 +103,15 @@ function Row(props) {
 
   const columnIndexOffSet = hasRowSelection ? 1 : 0;
 
-  const handleCellSelect = useCallback((rowIdColId, coordinates, cellSelectable = true) => {
-    if (hasRowSelection) {
+  const handleCellSelect = (selectionDetails) => {
```

Comment:
You removed the useCallback function on this method.  That needs to remain in place or it will cause unnecessary rerenders.

Reply:
Added back